packages/commonwealth/server/passport/magic.ts

```diff
@@ -140,7 +140,6 @@ async function createNewMagicUser({
   // completely new user: create user, profile, addresses
   return sequelize.transaction(async (transaction) => {
     const newUser = await models.User.createWithProfile(
-      models,
```

Comment:
I see that the usage of `createWithProfile` has changed, but the original function signature still requires `models` as the first arg?

Reply:
Nvm, I clicked the function in GH and it didn't show me the version on this branch.